core/common/scale/util.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "util.hpp"
+
+#include <boost/predef.h>
+#include <algorithm>
+#include <codecvt>
+#include <cstdint>
+#include <type_traits>
+
+#include "common/scale/types.hpp"
```

Comment:
```suggestion
#include <boost/predef.h>
#include "common/scale/types.hpp"
```

Reply:
You mean include order is wrong? Could you please specify the right order in terms file header, stl includes, boost includes, 3rd party includes?

Reply:
Now it looks like:
```suggestion
#include "common/scale/util.hpp"

#include <boost/predef.h>
#include <algorithm>
#include <codecvt>
#include <cstdint>
#include <type_traits>

#include "common/scale/types.hpp"
```